jupyterhub/templates/proxy/deployment.yaml

```diff
@@ -67,13 +67,14 @@ spec:
         - name: chp
           image: {{ .Values.proxy.chp.image.name }}:{{ .Values.proxy.chp.image.tag }}
           {{- $hubNameAsEnv := include "jupyterhub.hub.fullname" . | upper | replace "-" "_" }}
+          {{- $hubHost := printf "http://%s:$(%s_SERVICE_PORT)" (include "jupyterhub.hub.fullname" .) $hubNameAsEnv }}
```

Comment:
@consideRatio is there a reason not to use  {{$hubNameAsEnv}}_SERVICE_HOST since we use it for SERVICE_PORT?

Reply:
Yepp, IPv6 compatebility